file_writer_control/KafkaTopicUrl.py

```diff
@@ -0,0 +1,21 @@
+import re
+
+
+class KafkaTopicUrl:
+    """
+    Class for extracting address, port and topic name from a Kafka topic url.
+    """
+    test_regexp = re.compile(
+        r"^(\s*(kafka://)?((([^/?#:]+)+)(:(\d+){1,5})?)/([a-zA-Z0-9._-]+)\s*)$"
+    )
+
+    def __init__(self, url: str):
+        result = re.match(KafkaTopicUrl.test_regexp, url)
+        if result is None:
+            raise RuntimeError("Unable to match kafka url.")
+        self.port = 9092
```

Comment:
May be we can add a comment here stating that this is specific port used by Kafka and not an arbitrary choice for future reference.

Reply:
```suggestion
        self.port = 9092 # Default Kafka broker port
```
Sure.